File: examples/discord/jda/slash/src/main/java/dev/triumphteam/slash/example/ExampleCommand.java
package dev.triumphteam.slash.example;

import dev.triumphteam.cmd.core.annotations.Command;
import dev.triumphteam.cmd.slash.sender.SlashSender;
import net.dv8tion.jda.api.entities.User;

@Command("example")
public class ExampleCommand {

    @Command
    public void execute(final SlashSender sender, final String name, final User user) {

    }
}

File: examples/discord/kord/slash/src/main/kotlin/dev/triumphteam/kord/example/commands/ExampleCommand.kt
package dev.triumphteam.kord.example.commands

import dev.kord.core.entity.Attachment
import dev.kord.core.entity.Member
import dev.kord.core.entity.User
import dev.triumphteam.cmd.core.annotations.Command
import dev.triumphteam.cmd.discord.annotation.NSFW
import dev.triumphteam.cmds.kord.sender.SlashSender

@NSFW
@Command("example")
public class ExampleCommand {
    @Command
    public suspend fun execute(sender: SlashSender, member: Member, user: User, attachment: Attachment) {
        sender.reply("Command sent was /example <${member.username}> <${user.username}> <${attachment.filename}>")
    }
}
